mw-webapp/src/pages/page404/Page404.tsx

```diff
@@ -3,7 +3,11 @@ import styles from "src/pages/page404/Page404.module.scss";
 
 const ERROR_404 = "404 NOT FOUND";
 
-export const Page404 = () => {
+/**
+ * Page 404
+ * @returns {JSX.Element}
```

Comment:
Why do you use @returns? Doesn't your ide show types?

Reply:
Deleted